src/mgmt/services/manager/project/api/nodes.py

```diff
@@ -9,31 +9,196 @@
+                                                                   version='v1',
+                                                                   plural='droplets',
+                                                                   namespace='default', 
+                                                                   name=droplet['metadata']['name'],
+                                                                   body=droplet)
+        logger.info('Response for update droplet: {}'.format(update_response))
+    except ApiException as e:
+        logger.error('Exception when updating exsiting droplet: {}'.format(e))
+
+def delete_droplet(name):
+    try:
+        delete_response = obj_api.delete_namespaced_custom_object(group='zeta.com',
+                                                                  version='v1',
+                                                                  namespace='default',
+                                                                  plural='droplets', 
+                                                                  name=name,
+                                                                  body=client.V1DeleteOptions(),
+                                                                  propagation_policy="Orphan")
+        logger.info('Response for delete droplet: {}'.format(delete_response))
+    except ApiException as e:
+        logger.error('Exception when deleting droplet: {}\n{}'.format(name, e))
+
+def create_droplet(name, ip, mac, itf, network, zgc_id):
+    droplet_body = dict()
+    meta_data = dict()
+    meta_data['name'] = name
+    meta_data['labels'] = dict()
+    meta_data['labels']['zgc_id'] = zgc_id
+    meta_data['labels']['network'] = network
+    spec = dict()
+    spec['ip'] = ip
+    spec['mac'] = mac
+    spec['status'] = 'Init'
+    spec['itf'] = itf
+    spec['network'] = network
+    spec['zgc_id'] = zgc_id
+    droplet_body['metadata'] = meta_data
+    droplet_body['spec'] = spec
+    droplet_body['apiVersion'] = 'zeta.com/v1'
+    droplet_body['kind'] = 'Droplet'
+    try:
+        create_response = obj_api.create_namespaced_custom_object(group='zeta.com',
+                                                                  version='v1',
+                                                                  namespace='default',
+                                                                  plural='droplets', 
+                                                                  body=droplet_body)
+        logger.info('Response for create droplet: {}'.format(create_response))
+    except ApiException as e:
+        logger.error('Exception when Creating droplets: {}'.format(e))
+
 @nodes_blueprint.route('/nodes', methods=['GET', 'POST'])
 def all_nodes():
     if request.method == 'POST':
         post_data = request.get_json()
         post_data['node_id'] = str(uuid.uuid4())
+        response_object = deepcopy(post_data)
```

Comment:
Don't see any reason to make this change

Reply:
fixed